sklearn/utils/sparsefuncs.py

```diff
@@ -25,6 +26,49 @@ def _raise_error_wrong_axis(axis):
         raise ValueError(
             "Unknown axis value: %d. Use 0 for rows, or 1 for columns" % axis)
 
+def csr_csc_min_axis0(X):
```

Comment:
I'm curious, is there a performance problem with simply calling `X.min(axis=0)` and `X.max(axis=0)`? At first glance the code invoked there seems efficient. Are these functions faster?